public/scss/partials/_imageset-demo.scss

```diff
@@ -1,11 +1,19 @@
 .imageset-demo {
+	display: flex;
+	justify-content: space-between;
+	flex-direction: row;
+	flex-flow: wrap;
```

Comment:
I think `flew-flow` is shorthand for `flew-direction` & `flew-wrap`, meaning we can remove `flew-direction: row;` from the line above and change this line to be `flew-flow: row wrap;`